cli/src/pcluster/templates/login_nodes_stack.py

```diff
@@ -109,6 +111,10 @@ def _add_login_nodes_pool_launch_template(self):
             f"LoginNodeLaunchTemplate{self._pool.name}",
             launch_template_name=f"{self.stack_name}-{self._pool.name}",
             launch_template_data=ec2.CfnLaunchTemplate.LaunchTemplateDataProperty(
+                block_device_mappings=self._launch_template_builder.get_block_device_mappings(
```

Comment:
Let's add tests to confirm the expected configuration are applied to the template.

Reply:
Okay!

Reply:
Done! Just found there were some incorrect code in previous unit test. Have corrected them. All the modification also done for release-3.7 branch. BTW, thank you for helping me correct the PR title.